README.md

```diff
@@ -62,6 +62,10 @@ Here is an example for the dialplan:
     exten => _X.,1,Dial(Dongle/r1/${EXTEN})
     exten => _X.,n,Hangup
 
+    exten => _X.,n,DongleSendSMS(dongle0,${EXTEN},"Please call me",1440,yes)
+
+    exten => _X.,n,DongleSendUSSD(dongle1,*#123#)
+
```

Comment:
More complete in [othersipserver-incoming]:

```
exten => *#123#,1,DongleSendUSSD(dongle0,${EXTEN})
exten => *#123#,n,Answer()
exten => *#123#,n,Wait(2)
exten => *#123#,n,Playback(vm-goodbye)
exten => *#123#,n,Hangup()

exten => _#X.,1,DongleSendSMS(dongle0,${EXTEN:1},"Please call me",1440,yes)
exten => _#X.,n,Answer()
exten => _#X.,n,Wait(2)
exten => _#X.,n,Playback(vm-goodbye)
exten => _#X.,n,Hangup()
```

Is there an easy way to change my last commit / push (with git linux console)?